### ISSUE TITLE
Location strings incorrect for Hush and Push and Lazarus

### ISSUE THREAD

**Issue by TonyFlury** [jixxed/ed-odyssey-materials-helper#71]
I have a really bad Hush and Push habit in game (I like the cash), but the location info for both is incorrect.

The in-app info implies that both Hush and Push are availble Skimmer(Irregular Marker) and SRV(Irregular Marker), 

The Location info for Lazarus says that it is avaialbe in Sealed Lockers in Argicultural, Industrial, Laboratory and Medical settlements.

In fact all 3 are available in Anarchy systems in ALL settlement types (for instance I find large quantities of all 3 in Tourist settlments), and in Smuggler Caches (Irregular Markers). 

None of them are available from Skimmers or SRVs.

**Comment by jixxed** [comment]
What version of the app are you on? Lockers haven't been listed in a while. Your version is probably still based on manually collected data, while the recent version uses automatically collected data.

**Comment by TonyFlury** [comment]
You are correct my apologies - I had downloaded it and thought I had installed it.
Apologies again - great application.
